prow/cluster/deck_private_deployment.yaml

```diff
@@ -30,10 +30,21 @@ spec:
         - --redirect-http-to=prow-private.istio.io
         - --kubeconfig=/etc/kubeconfig/istio-config
         - --gcs-credentials-file=/etc/service-account/service-account.json
-        - --github-token-path=/etc/github/oauth
         - --spyglass
         - --hidden-only
+        - --rerun-creates-job
+        - --oauth-url=/github-login
+        - --github-token-path=/etc/github/oauth
+        - --github-endpoint=http://ghproxy
```

Comment:
This flag should be used twice to specify the upstream API URL as a fallback for when ghproxy in unavailable: https://github.com/istio/test-infra/search?q=--github-endpoint&type=Code

Reply:
Missed that, thanks. Updated